src/AutoPilotPlugins/PX4/PX4SimpleFlightModes.qml

```diff
@@ -193,6 +193,26 @@ Item {
                                     indexModel: false
                                 }
                             }
+
+                            Row {
+                                spacing: ScreenTools.defaultFontPixelWidth
+
+                                property Fact fact: controller.getParameterFact(-1, "RC_MAP_TRANS_SW")
```

Comment:
This will still pop a missing parameter error. Take a look at APMSensorsComponent.qml for examples of how to structure it to prevent that.